README.md

```diff
@@ -197,8 +103,8 @@ Build the image locally using this git repository as the [build context](https:/
 ```console
 docker build \
   --build-arg VERSION=0.0.1 \
-  --tag cisagov/example:0.0.1 \
-  https://github.com/cisagov/example.git#develop
+  --tag cisagov/admiral-docker:0.0.1 \
```

Comment:
Use correct image name.  Also, versions in the README should match the value in src/version.txt:
```suggestion
  --tag cisagov/admiral:1.3.0 \
```